openebs/ci/snapshot/openebs-operator.yaml

```diff
@@ -0,0 +1,189 @@
+# Define the Service Account
```

Comment:
How to keep this file in sync with the actual openebs-operator.yaml? Is this fine really required? Why not install via helm?

Reply:
With helm there is issue to get the OPENEBS_MAPI_SERVICE(its auto generated prefix name) env to get MAPI_ADDR , which is set during the openebs installation using helm( inside provisioner POD), and can not be imported in snapshot-controller and snapshot-provisioner pod

Reply:
openebs-operator yaml file uses `ci` images of maya-apiserver, provisioner and jiva.

Reply:
Can we use the following:
```
helm install stable/openebs --name ci --namespace openebs --set apiserver.imageTag="ci",jiva.replicas="1",jiva.imageTag="ci", provisioner.imageTag="ci"
```

The MAPI_ADDR can then be obtained using:
```
MAPI_SVC_ADDR=`kubectl get service -n openebs ci-openebs-apiservice -o json | grep clusterIP | awk -F\" '{print $4}'`
```

It will be good to avoid duplicating the operator.yaml files. Infact we should reduce the usage so much and then remove it. The operator-yaml should be auto-generated from helm chart.

Reply:
How we can export this inside a pod containers ?